### DISCUSSION TITLE
Welcome to video-app-agora-rtc Discussions!

### DISCUSSION THREAD

**Discussion by sunny7899** [angulardevelopment/video-app-agora-rtc#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rozarlive** [comment]
@sunny7899 I saw on the youtube video there is a RTM service for , was that taken out of the project? I tested the repo and it is working great with the video. Great work btw.

**Comment by sunny7899** [comment]
Yes working on another project. In that you will get both rtc and rtm packages info.

**Reply by rozarlive** [reply]
> Yes working on another project. In that you will get both rtc and rtm packages info.

@sunny7899 great to hear. I can't wait to try it. I hope it will be soon. And thank you for the reply.
